### PR TITLE
completed week1 nodeJs assignment

### PR DESCRIPTION
**Pull request by shreyansh-khatri**
Pull Request Questionnaire

1) Briefly describe your change.
    Completed NodeJs week1 daily as well as weekly assignments. 

2) What parts of the application are affected by your change?
    My changes only affected the week1 branch of this repository

3) How did you test your change?
    I tested various parts such as validation, different inputs and some other edge cases.

### CODE REVIEW

File: src/Week1/dailyAssignments/assignment1.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
+const {
+  task1Message,
+  task2Message,
+}: { task1Message: string; task2Message: string } = require("../utils.ts");
+
+async function task1(): Promise<string> {
```
Comment (aditya-s-19) [inline src/Week1/dailyAssignments/assignment1.ts]: use arrow function variables

File: src/Week1/dailyAssignments/assignment3.ts [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  notFound: string;
+  invalidId: string;
+  portMessage: string;
+  deleteMessage: string;
+} = require("../utils.ts");
+const app = express();
+app.use(express.json());
+const PORT: number = parseInt(process.env.PORT || "3000");
+
+const brands = [
```
Comment (aditya-s-19) [inline src/Week1/dailyAssignments/assignment3.ts]: make an interface for this

File: src/Week1/utils.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+// module.exports = {
+//   notFound,
+//   alreadyExists,
+//   task1Message,
+//   task2Message,
+//   errorMessage,
+//   message,
+//   invalidId,
+//   portMessage,
```
Comment (aditya-s-19) [inline src/Week1/utils.ts]: remove these comments

File: src/Week1/dailyAssignments/assignment3.ts [hunk 4]
```diff
@@ -0,0 +1,94 @@
+import express, { Request, Response } from "express";
+import Joi from "joi";
+
+interface Brand {
+  id: number;
+  name: string;
+}
+
```
Comment (RitikaGarg-ITT) [inline src/Week1/dailyAssignments/assignment3.ts:5]: interfaces in different files

**Review by choudhary-keshav — APPROVED** [review]

**Review by RitikaGarg-ITT — APPROVED** [review]

**Review by aditya-s-19 — APPROVED** [review]

**Review by shreegovinda — APPROVED** [review]
